app/enrichment/validate-values.js

```diff
@@ -1,7 +1,9 @@
 const validateValues = (value, invoiceLines) => {
   const lineValues = invoiceLines.reduce((x, y) => x + y.value, 0)
   if (lineValues !== value) {
-    throw new Error(`Payment request is invalid. Invoice line values (${lineValues}) do not match header (${value})`)
+    const error = new Error(`Payment request is invalid. Invoice line values (${lineValues}) do not match header (${value})`)
```

Comment:
Does this also need a `cause` for consistency?

Reply:
`cause` is for building up a stack trace of errors so if one error causes another you can chain them.  This is the first error thrown in this chain so there is no `cause` to attach.